fitnessllm_dataplatform/batch_handler.py

```
@@ -74,0 +1,101 @@
+
+    @beartype
+    def process_user(self, user_id: str, data_source: str = "strava") -> None:
+        """Process a single user.
+
+        Args:
+            user_id: User ID from Firestore.
+            data_source: Data source to process (default: strava).
+        """
+        try:
+            structured_logger.info(
+                message="Processing user", uid=user_id, data_source=data_source
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)